### DISCUSSION TITLE
ESC only works if hit twice under GNU Screen

### DISCUSSION THREAD

**Discussion by jaredkrinke** [helix-editor/helix#6733] [Q&A] [answered]
I'm aware of the FAQ entry about ESC being delayed under Screen/tmux, but even with "maptimeout 0" set in Screen, in Helix I have to hit ESC twice in a row for it to work. It works fine outside Screen.

For now, I'm mapping Ctrl-c to various "exit" commands, but has anyone else has run into this?

**Comment by rhogenson** [comment, marked as answer]
I had the same issue. I believe this is a bug in screen: https://savannah.gnu.org/bugs/?57748

Updating screen to version 5.0.0 fixed the issue for me
